src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -53,32 +50,48 @@ public ReadsSparkSource(JavaSparkContext ctx) {
 
 
     /**
-     * Loads Reads using Hadoop-BAM. For local files, bam must have the fully-qualified path,
+     * Loads Reads using Hadoop-BAM. For local files, readFileName must have the fully-qualified path,
      * i.e., file:///path/to/bam.bam.
-     * @param bam file to load
+     * @param readFileName file to load
+     * @param reference Reference file or null if not available. Reference is required for CRAM files.
      * @param intervals intervals of reads to include.
      * @return RDD of (SAMRecord-backed) GATKReads from the file.
      */
-    public JavaRDD<GATKRead> getParallelReads(final String bam, final List<SimpleInterval> intervals) {
-        return getParallelReads(bam, intervals, DEFAULT_SPLIT_SIZE);
+    public JavaRDD<GATKRead> getParallelReads(final String readFileName, final File reference, final List<SimpleInterval> intervals) {
+        return getParallelReads(readFileName, reference, intervals, DEFAULT_SPLIT_SIZE);
```

Comment:
What will happen if you call this method with a null reference and a cram input?